src/main/java/hudson/plugins/parameterizedtrigger/BuildInfoExporterAction.java

```diff
@@ -243,40 +380,48 @@ public void buildEnvVars(AbstractBuild<?, ?> build, EnvVars env) {
    * @return
    */
   public Object readResolve() {
-    if (this.lastReference == null) {
-      this.lastReference = new BuildReference(this.buildName, this.buildNumber, Result.NOT_BUILT);
+    if (this.lastReference2 == null) {
+      this.lastReference2 = this.lastReference;
```

Comment:
Does this ever have a chance to get set to anything?  It appears that all the changes now run through `lastReference2` so this potentially never gets set unless I'm missing something.